schema/src/main/scala/io/shiftleft/codepropertygraph/schema/Method.scala

```diff
@@ -40,6 +40,21 @@ object Method extends SchemaBase {
       )
       .protoId(22)
 
+    val varArgParameterIndex = builder
+      .addProperty(
+        name = "VARARG_PARAMETER",
+        valueType = ValueTypes.INTEGER,
+        cardinality = Cardinality.One,
+        comment =
+          """
+            |For variadic methods the VARARG_PARAMETER property specifies which parameter
```

Comment:
This is a bit weird because it means that the `METHOD` node stores information about another node, namely, the parameter. Any reason why we don't have an `IS_VARIADIC` field in the parameter instead?